compoments/titledRadioFieldSet.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RadioFieldSet from './radioFieldSet';
+
+const TitledRadioFieldSet = ({name, title, handleChange, values, options}) => {
```

Comment:
This component seems a bit obsolete

Reply:
The `Continue` was duplicated in the 2 components I had using `RadioFieldSet` and this component removes that duplication.

Reply:
As mentioned in my PR comment, it was to reduce duplication
> I [extracted](https://github.com/City-of-Lincoln-Council/housing-repairs-online-frontend/pull/14ced6adee3ffc6bd090414ffc406383774e0858) a common control, `titledRadioFieldSet` to remove some duplication.

It's captured in a single [commit](https://github.com/City-of-Lincoln-Council/housing-repairs-online-frontend/pull/14ced6adee3ffc6bd090414ffc406383774e0858), meaning it can be reverted easily.
Let's discuss the best approach.

Reply:
Agreed to remove `TitledRadioFieldSet`